L1Trigger/TrackFindingTracklet/python/ProducerKF_cfi.py

```diff
@@ -13,6 +14,8 @@
   BranchLostStubs      = cms.string  ( "StubLost"      ),                                        #
   BranchLostTracks     = cms.string  ( "TrackLost"     ),                                        #
   CheckHistory         = cms.bool    ( False ),                                                  # checks if input sample production is configured as current process
-  EnableTruncation     = cms.bool    ( True  )                                                   # enable emulation of truncation, lost stubs are filled in BranchLost
+  EnableTruncation     = cms.bool    ( True  ),                                                  # enable emulation of truncation, lost stubs are filled in BranchLost
 
+  SummerChain         = cms.bool  ( False ),                                                    # reduce l1 tracking to summer chain configuration
```

Comment:
The summer chain will only exist for a few more months. So we should minimize and localise as much as possible any code that is specific to the summer chain. And try to generalize it so it will work with larger chains, which we'll soon be testing.
Also, it's messy if we need to set SummerChain=true here and in Settings.h
Also, do SummerChainChannels represent which input channels of Apollo are connected to DTC? If so, why is this in the KF config file? Where do these numbers come from?

Reply:
I can't easily retrieve the Reduced bool from Settings in other ED or ES produces as L1FPGATrackProducer. Using an ESProducer to produce and manipulate Settings would be very handy. As long as that is the case we will have to live with duplicated settings. I can replace the name SummerChain with Reduced if that makes you happy. I can create a new cfi file for the vector of dtc channel if that makes you happy as well. I translated them by hand.